pom.xml

```diff
@@ -1541,9 +1542,7 @@
                     </execution>
                 </executions>
             </plugin>
-        </plugins>
-        <pluginManagement>
-            <plugins>
+
```

Comment:
Generating the project with `mvn eclipse:eclipse` doesn't seem to have made any difference.  I still get the errors about the plugin execution not covered by lifecycle configuration, and errors loading the property files.

Not sure if that's because my Eclipse has the m2e plugins installed, though.  I don't want to remove those since I need them for other projects.